### PR TITLE
Splits up `item_inhands.dmi`

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Splits up `item_inhands.dmi` and removes some unused sprites

**TODO:**
- Even more testing
- Bring the edagger back to the original file

## Why It's Good For The Game
Anytime anyone even adds an inhands sprite, it's put in `item_inhands.dmi` 90% of the time, causing mass conflicts any time any PR is merged for em. This aims to lessen the load a bit, and also reduce some lag while the game is loading in the entire sprite list.

## Images of changes
Nothing should be changed

## Testing
I spawned in a fuckton of items

## Changelog
Mass conflicts for Jimkil specifically

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1243,6 +1249,8 @@
 /obj/item/toy/chainsaw
 	name = "Toy Chainsaw"
 	desc = "A toy chainsaw with a rubber edge. Ages 8 and up"
```
Comment (Henri215) [inline code/game/objects/items/toys.dm:1251]:
```suggestion
	desc = "A toy chainsaw with a rubber edge. Ages 8 and up."
```
[insert "while we are here" here]

File: code/game/objects/items/weapons/weaponry.dm [hunk 2]
```diff
@@ -75,6 +77,8 @@
 /obj/item/katana
 	name = "katana"
 	desc = "Woefully underpowered in D20"
```
Comment (Henri215) [inline code/game/objects/items/weapons/weaponry.dm:79]:
```suggestion
	desc = "Woefully underpowered in D20."
```

File: code/modules/antagonists/vampire/vampire_powers/hemomancer_powers.dm [hunk 3]
```diff
@@ -40,6 +40,8 @@
 	name = "vampiric claws"
 	desc = "A pair of eldritch claws made of living blood, they seem to flow yet they are solid"
```
Comment (Henri215) [inline code/modules/antagonists/vampire/vampire_powers/hemomancer_powers.dm:41]:
```suggestion
	desc = "A pair of eldritch claws made of living blood, they seem to flow yet they are solid."
```

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 4]
```diff
@@ -216,6 +222,8 @@
 
 /obj/item/nullrod/claymore/saber/pirate
 	name = "nautical energy cutlass"
+	lefthand_file = 'icons/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons_righthand.dmi'
```
Comment (Henri215) [inline code/game/objects/items/weapons/holy_weapons.dm:226]: here too

File: code/game/objects/items/weapons/holy_weapons.dm [hunk 5]
```diff
@@ -245,6 +253,8 @@
 
 /obj/item/nullrod/scythe/vibro
```
Comment (Henri215) [inline code/game/objects/items/weapons/holy_weapons.dm:254]: Do you consider the /scythe not to be a weapon?

File: code/game/objects/items/weapons/melee/melee_misc.dm [hunk 6]
```diff
@@ -101,6 +103,8 @@
 	name = "spellblade"
 	desc = "An enchanted blade with a series of runes along the side."
 	icon = 'icons/obj/guns/magic.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons_righthand.dmi'
```
Comment (Henri215) [inline code/game/objects/items/weapons/melee/melee_misc.dm:107]: inherits from /melee

File: code/game/objects/items/weapons/powerfist.dm [hunk 7]
```diff
@@ -1,6 +1,8 @@
 /obj/item/melee/powerfist
 	name = "power-fist"
 	desc = "A metal gauntlet with a piston-powered ram ontop for that extra 'ompfh' in your punch."
+	lefthand_file = 'icons/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons_righthand.dmi'
```
Comment (Henri215) [inline code/game/objects/items/weapons/powerfist.dm:5]: inherits from /melee

File: code/modules/surgery/organs/augments_arms.dm [hunk 8]
```diff
@@ -466,6 +466,8 @@
 /obj/item/shield/v1_arm
 	name = "vortex feedback arm"
 	desc = "A modification to a users arm, allowing them to use a vortex core energy feedback, to parry, reflect, and even empower projectile attacks. Rumors that it runs on the user's blood are unconfirmed."
+	lefthand_file = 'icons/mob/inhands/weapons_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons_righthand.dmi'
```
Comment (Henri215) [inline code/modules/surgery/organs/augments_arms.dm:470]: inherits from /shield

**Comment by Henri215** [comment]
image
image
Different amount of sprites between left and right item inhand files.

**Comment by DGamerL** [comment]
> image image Different amount of sprites between left and right item inhand files.

I think I know what that is, I'll see if I can fix that

**Review by Henri215 — APPROVED** [review]
it is looking good to me

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**